### PR TITLE
DB 1476 Help Page Service Desk

### PR DESCRIPTION
**Pull request by kimminahm**
Added page content for Help Page to go to the service desk

### CODE REVIEW

File: src/js/components/help/helpContent.jsx [hunk 1]
```diff
@@ -78,6 +78,14 @@ export default class HelpContent extends React.Component {
                 If you have questions about Broker access, or if your agency is not listed on the MAX page linked above, email <a href='mailto:DATAPMO@fiscal.treasury.gov?subject=Broker%20Access%20Information' target='_top'>DATAPMO@fiscal.treasury.gov</a>.
             </p>
             </div>;
+        }else{
+            membership = <p>
+                If you encounter a bug, have a question, or need help, 
+                please register for the <a href='https://servicedesk.usaspending.gov'>USAspending Service Desk</a> and submit a ticket.
```
Comment (alburde1) [inline src/js/components/help/helpContent.jsx]: Do we want to throw in a `target="_blank"` here so it's like all the others?

File: src/js/components/help/helpContent.jsx [hunk 2]
```diff
@@ -78,6 +78,14 @@ export default class HelpContent extends React.Component {
                 If you have questions about Broker access, or if your agency is not listed on the MAX page linked above, email <a href='mailto:DATAPMO@fiscal.treasury.gov?subject=Broker%20Access%20Information' target='_top'>DATAPMO@fiscal.treasury.gov</a>.
             </p>
             </div>;
+        }else{
```
Comment (alburde1) [inline src/js/components/help/helpContent.jsx]: Generally, I believe our else statements are on a new line with a space before the `{` (I'm already asking for something else, might as well fix this while you're there)

File: src/js/components/help/helpContent.jsx (hunk 1 above)
Comment (kimminahm) [inline src/js/components/help/helpContent.jsx]: fixed

File: src/js/components/help/helpContent.jsx (hunk 2 above)
Comment (kimminahm) [inline src/js/components/help/helpContent.jsx]: fixed

**Pull request merged**